src/main/java/org/checkerframework/specimin/UnsolvedSymbolVisitor.java

```diff
@@ -271,6 +276,15 @@ public void setExceptionToFalse() {
     gotException = false;
   }
 
+  @Override
+  public Node visit(ImportDeclaration decl, Void arg) {
+    if (decl.isAsterisk()) {
+      throw new RuntimeException(
+          "A wildcard import statement found. Please use explicit import" + " statements.");
```

Comment:
Rather than "please use", which sounds like an order, it would be better for this error message to indicate that **we** don't support wildcard imports. I suggest the following text:
"Specimin does not support wildcard import statements, but a wildcard import was found in " + the path to the file.